usr/src/man/man1m/zdb.1m

```diff
@@ -217,6 +220,18 @@ three times display every spacemap record.
 .sp
 .ne 2
 .na
+\fB-o\fR \fI"ZFS plain file"\fR \fIdataset\fR \fIfilepath\fR
```

Comment:
Well, I figured this option might come in handy for other lookup types, and since the strings already exist in dmu_ot, it seemed easiest to just use those.  Granted, not quite so easy from a typing point of view.